warmup/summary_statistics.py

```diff
@@ -124,12 +124,7 @@ def collect_summary_statistics(data_dictionaries, half_bound, delta, steady_stat
                     else:
                         error_iter = (error_iter,)
                     median_time_to_steady, error_time_to_steady = bootstrap_confidence_interval(time_to_steadys)
-                    # Check whether the error is wider than the median, OR
-                    # whether the lower CI of the time to reach a steady state
-                    # implies that one or more of the early iterations took
-                    # < 0 seconds to complete.
-                    if (error_time_to_steady > median_time_to_steady or
-                        (median_time_to_steady - error_time_to_steady) / median_iter < 1.0):
+                    if error_time_to_steady > median_time_to_steady:
```

Comment:
Good question. If the steady iter mean is 2, and the error is 2, then the lower bound of the ci is 0. That probably makes sense for steady iter and the time to reach a steady state. I'm not sure that it makes sense to say that the lower bound of the steady state perf is 0, as that would suggest that an iteration can complete in 0s.